### ISSUE TITLE
Incorrect name is provided for "Add Task" button as "Add": A11y_.NET 9 template_.NET 9 template_Project_Name Role Value.

### ISSUE THREAD

**Issue by AnKushSingh05** [dotnet/maui#27734]
### GitHub Tags
#A11yTCS; #A11ySev2; #Win32; #WCAG4.1.2; #GH_.NET9template_Win32_Apr2024; #DesktopApp; #.NET Core; #; #A11yMAS; #Name Role Value;

### Environment Details:
Application Name: .NET 9 template tests that include some 3rd party controls
OS: Windows 11 Enterprise 24H2 (OS build 26100.2314)

### Repro Steps:

1. Install and open ".NET 9 template" app.
2. "Home Screen" will open.
3. Tab till new button.
4. Observe that Incorrect name is provided for "Add Task" button as "Add"

### Actual Result:
Incorrect name is provided for "New Task" button as "Add"

### Expected Result:
Correct name as "Add task" should be provided for button.

### User Impact:
Screen reader users will not get the information for which Add button is provided.

### Attachment:

![Image](https://github.com/user-attachments/assets/6f083f06-0bf9-484d-907e-20956a811c31)

**Comment by AnKushSingh05** [comment]
Closing the bug as issue is fixed.

![Image](https://github.com/user-attachments/assets/35416c10-161b-4b32-8760-3c08dad43b19)
